bgpd/bgp_route.c

```diff
@@ -15778,6 +15778,30 @@ static int bgp_distance_unset(struct vty *vty, const char *distance_str,
 	return CMD_SUCCESS;
 }
 
+int bgp_address_family_distance_clean()
```

Comment:
Please use the same pattern as `bgp_static_delete()`. More/less identical except for some adjustments.

Reply:
Done

Reply:
I meant not only `void()`, but the structure, macros, logic inside also.